frontend/pages/recipes/all.vue

```diff
@@ -62,6 +65,10 @@ export default defineComponent({
       title: this.$t("page.all-recipes") as string,
     };
   },
+  methods: {
+    assignSorted(val: Array<Recipe>) {
+      this.recipes = val;
+    },
+  },
```

Comment:
Two things here.

1. We're trying to consolidate all functionality to the composition API, can you implement this function is the setup() function and return it to make it available to the template?
2. I took this out because I don't think the sorting does what's expected. At the moment, sort is handled on the client side, so you'll only sort the items that have been fetched. I'd expect that sorting would refetch the recipes and utilize sorting on the backend... which isn't really implemented yet. My feeling is that if we implement it this way, it's just going to cause issues to be filed.

Reply:
Got it, I can move the function to setup. I can also just un-restore the sort if you think it'll cause issues, let me know. It seems fine on my instance with 121 recipes, but maybe larger lists will have issues:
![example](https://user-images.githubusercontent.com/71845777/173917686-a5b27015-70b9-46f1-bc5a-7f7b4533469a.gif)

But I'm happy to ditch it since we're reworking it to be server side anyway. I think we should still fix the infinite scroll issue, though

Reply:
Actually scratch that, I already found some issues with it, lol